### PR TITLE
Support other platforms

### PR DESCRIPTION
**Pull request by trombik**
initial support of other platforms. there would be bugs for sure. but every distributions manages daemons differently, gave up for now.

### CODE REVIEW

File: tests/serverspec/default_spec.rb [hunk 1]
```diff
@@ -45,6 +47,43 @@
...
+    it { should be_grouped_into default_group }
+    its(:content) { should match(/^OPTIONS="-w"$/) }
+  end
+
+  describe service("rpcbind") do
+    it { should be_enabled }
+    it { should be_running }
+  end
+
+  puts os[:release]
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb]: puts debug should be removed.

File: tasks/configure-OpenBSD.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+
+# XXX rpc.lockd(8) of OpenBSD does not support client-side of the protocol
```
Comment (mitsururike) [inline tasks/configure-OpenBSD.yml:3]: if this comment means `# rpc.lockd is supposed to be configured here. However,`,  it should be written explicitly.

**Review by mitsururike — APPROVED** [review]

File: tasks/configure-OpenBSD.yml (hunk 2 above)
Comment (trombik) [reply tasks/configure-OpenBSD.yml:3]: these days, most, if not all, of distributions can `mount -t nfs` by default. what you would expect an NFSv3 role to do is to configure locking. you do not need an `ansible` role to mount NFS file system. if you do not need locking, there is no reason to use the role. what else should be commented?

Comment (mitsururike) [reply tasks/configure-OpenBSD.yml:3]: nothing else but just i could not understand why the comment is written here without reading other configure-*.yml.

Comment (trombik) [reply tasks/configure-OpenBSD.yml:3]: the fact that NFS lock does not work in OpenBSD will be documented in README, then.

**Pull request merged**